src/archetypes.cpp

```diff
@@ -396,7 +396,6 @@ struct archetype_data *generate_decker() {
 
   // Inventory.
   arch->weapon = OBJ_SCK_MODEL_100;
-  arch->weapon_barrel = OBJ_GAS_VENT_IV;
```

Comment:
Why are we removing the vent IV?

Reply:
Added the gas vent IV to the new firearm itself (obj vnum 91000), along with a smartlink-2 to match the decker archetype cyberware selection (it was previously a standard smartlink accessory).